index.html

```diff
@@ -5,11 +5,92 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pencil-ui.netlify.app/css/main.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+        rel="stylesheet">
+        <link rel="stylesheet" href="./style.css">
     <title>eBuy</title>
 </head>
+<style>
+    * {
+        font-family: 'Raleway', sans-serif;
+    }
+</style>
 
 <body>
 
 </body>
+<div class="home_container">
+    <div class="navigation flex shadow">
```

Comment:
Try making use of semantic HTML tags like `nav`, `section`, `main`, `header`, `footer` etc for readability purposes as well as accessibility instead of div everywhere and use `div` only for smaller chunks inside the semantic tags.

Reply:
mostly things are reusable components now, so navbar I am using from CL, so there it has been used as nav

Reply:
I don't really understand what you're trying to say, can you elaborate? According to my understanding, you are using the navbar from your component library where you are using `nav` so then if that's the case then here also nav should be used? Correct me if I am wrong in my understanding?